dojo/tools/checkmarx/parser.py

```diff
@@ -119,8 +119,9 @@ def process_result_file_name_aggregated(self, dupes, findingdetail, query, resul
             find = Finding(title=title,
                            cwe=int(cwe),
                            test=self.test,
-                           active=False,
-                           verified=False,
+                           # These lines are commented so the default active/verified will not be set as False (Relate to issue #4598)
```

Comment:
All 3 lines with comments should go completely. With the git versioning and the link to issue and pull request there is all the information why the lines have been deleted.

Reply:
Agreed